cypress/utils/cookies.ts

```diff
@@ -0,0 +1,6 @@
+import { BACKEND_URL } from "../fixtures/constants"
+
+export const setCookieWithDomain = (name: string, value: string) => {
+  const baseUrl = new URL(BACKEND_URL)
+  cy.setCookie(name, value, { domain: baseUrl.host })
+}
```

Comment:
Nit: could we also have an optional param for options, to keep with the inbuilt functionality that setCookie has?

Reply:
added